.github/workflows/build-android.yml

```
@@ -30,6 +30,19 @@ jobs:
         with:
           target: ${{ matrix.target }}
 
+      - name: setup android sdk
+        uses: android-actions/setup-android@v3
+        with:
+          # Don't download unnecessary packages.
+          # We only need sdkmanager for the next step.
+          packages: ''
+
+      - name: install ndk and set up environment
+        run: |
+          ndk_version=$(cat crypto-ffi/bindings/android/ndk.version)
```

I forgot the syntax earlier, but strictly speaking we don't need `cat` here, as [bash knows how to read files into an env var directly](https://stackoverflow.com/a/10771857):

```suggestion
          ndk_version=$(<crypto-ffi/bindings/android/ndk.version)
```